docs: document new helm repository related k-cli commands

vespian opened the pull request:
## Jira Ticket

<!-- Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel. -->

<!-- Link to JIRA ticket -->
https://jira.d2iq.com/browse/D2IQ-83457

## Description of changes being made
Document new helm repository related k-cli commands. These changes are for upcoming 2.2 release. There is no 2.2 directory for kommander yet so I edited the 2.1 docs. Please feel free to adjust/change this PR however you like. 

## Checklist
- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

cmastr commented on pages/dkp/kommander/2.1/install/air-gapped/index.md:
```diff
@@ -26,6 +28,52 @@ Before installing, ensure you have:
 
+### Kommander charts bundle
+
+Charts bundle is a Gziped Tar archive containing Helm charts required during Kommander installation.
```
Need to add this to accept.txt

Colin1841 replied:
Added to the list: (Gzipped|gzipped)

cmastr commented on pages/dkp/kommander/2.1/install/air-gapped/index.md:
```diff
@@ -26,6 +28,52 @@ Before installing, ensure you have:
+
+Charts bundle is a Gziped Tar archive containing Helm charts required during Kommander installation.
+It can be created using Kommander CLI or downloaded together with Kommander CLI
+In order to create bundle please execute command:
```
```suggestion
To create a bundle:
```

Colin1841 replied:
```suggestion
Execute this command to create the charts bundle:
```

cmastr commented on pages/dkp/kommander/2.1/install/air-gapped/index.md:
````diff
@@ -26,6 +28,52 @@ Before installing, ensure you have:
+    kommander helmmirror create bundle
+    ```
+
+This will result in file `charts-bundle.tar.gz` being created.
````
```suggestion
Kommander creates `charts-bundle.tar.gz`.
```

cmastr commented on pages/dkp/kommander/2.1/install/air-gapped/index.md:
````diff
@@ -26,6 +28,52 @@ Before installing, ensure you have:
+    ```
+
+This will result in file `charts-bundle.tar.gz` being created.
+Optionally, the output file can be specified using `-o` parameter:
````
```suggestion
Optionally, specify the output using the `-o` parameter:
```

cmastr commented on pages/dkp/kommander/2.1/install/air-gapped/index.md:
```diff
@@ -26,6 +28,52 @@ Before installing, ensure you have:
+
+### Kommander's internal Helm repository
+
+Kommander charts bundle is uploaded to Kommander's internal Helm repository.
```
```suggestion
The Kommander charts bundle is uploaded to Kommander's internal Helm repository.
```

cneth commented on pages/dkp/kommander/2.1/install/air-gapped/index.md:
```diff
@@ -26,6 +28,52 @@ Before installing, ensure you have:
+### Kommander charts bundle
+
+Charts bundle is a Gziped Tar archive containing Helm charts required during Kommander installation.
+It can be created using Kommander CLI or downloaded together with Kommander CLI
```
```suggestion
It can be created with the Kommander CLI or downloaded along with the Kommander CLI.
```

msdolbey replied:
Create the charts bundle with the Kommander CLI or download along with the Kommander CLI.

cneth commented on pages/dkp/kommander/2.1/install/air-gapped/index.md:
```diff
@@ -26,6 +28,52 @@ Before installing, ensure you have:
+### Kommander's internal Helm repository
+
+Kommander charts bundle is uploaded to Kommander's internal Helm repository.
+It contents can be inspected using following command:
```
```suggestion
To inspect the contents:
```

cneth commented on pages/dkp/kommander/2.1/install/air-gapped/index.md:
````diff
@@ -26,6 +28,52 @@ Before installing, ensure you have:
+    kommander helmmirror upload chart [chartTarball]
+    ```
+
+Or upload new bundle altogether:
````
```suggestion
Or upload a new bundle:
```

Colin1841 commented on pages/dkp/kommander/2.1/install/air-gapped/index.md:
```diff
@@ -17,6 +17,8 @@ Before installing, ensure you have:
 
 - A Docker registry containing all the necessary Docker installation images, including the Kommander images. The `kommander-image-bundle.tar` tarball has the required artifacts.
 
+- A charts bundle file containing all the Helm charts that Kommander installation needs.
```
```suggestion
- A charts bundle file containing all Helm charts that Kommander installation needs.
```

Colin1841 commented on pages/dkp/kommander/2.1/install/air-gapped/index.md:
```diff
@@ -26,6 +28,52 @@ Before installing, ensure you have:
 
+### Kommander charts bundle
+
+Charts bundle is a Gziped Tar archive containing Helm charts required during Kommander installation.
```
```suggestion
The charts bundle is a Gziped Tar archive containing Helm charts, which are required during Kommander installation.
```

vacas5 reviewed (changes requested):
the source branch for this should be `main`

cmastr commented:
> the source branch for this should be `main`

@vacas5 and @vespian the JIRA for this seems to be a 2.2 change. So develop should be the correct branch, but this needs to be in the 2.2 folder. Am I interpreting that correctly?

msdolbey commented:
Yeah, this looks like it's just a file that needs to be adjusted (in the 2.2 folder)

vespian commented:
Thank you for thorough review and the fixes. I addressed all the comments and added some extra information wrt. installation. Please have another look.
